### ISSUE TITLE
Function cannot be used outside of evalString

### ISSUE THREAD

**Issue by jeroenvandijk** [borkdude/sci#259]
**version**

"@borkdude/sci": "0.0.12",

**platform**

Javascript, NodeJS, 

**problem**

Functions created in `evalString` cannot be used in the javascript context 

**repro**

```js
const { evalString } = require('@borkdude/sci');
var myFN = evalString("(fn [a] (+ a 10))")
myFN(4)
```
Gives

```
myFN(4)
      ^
TypeError: myFN is not a function
```

**expected behavior**

evalString should return a function that can be used from the javascript context. Like in the CLJ version:
```
(let [f (sci/eval-string "(fn [a] (+ a 10))")]
  (f 4))
```
Code above returns 14

**Comment by jeroenvandijk** [comment]
The function can be retrieved by accessing fields of the returned value. E.g. 
```
 var f = evalString("(clj->js (fn [a b] (+ a b)))")
  console.log(f.s(1, 2))
```

I'm wondering if this is because of the metadata that is added: https://github.com/borkdude/sci/blob/c2ffce590f8185b9fd5fdbb7e3746ba37f2ae02f/src/sci/impl/fns.cljc#L84 For all none macro fn's this is not needed I think

**Comment by borkdude** [comment]
@jeroenvandijk 

```
const { evalString, toJS } = require('@borkdude/sci');
```

and then call `toJS` to the function, this will remove the metadata.

**Comment by borkdude** [comment]
I'll check if non-macro functions work correctly without metadata and then publish a new release.

**Comment by borkdude** [comment]
Fixed in npm version 0.0.13-alpha.5
